app/controllers/webhook_controller.rb

```diff
@@ -23,12 +23,38 @@ def callback
       when Line::Bot::Event::Follow
         line_user_id = event['source']['userId']
         if !Customer.exists?(line_user_id: line_user_id) && line_user_id.present?
-          name = LineApi.get_profile(line_user_id)['displayName']
-          Customer.create!(name: name, line_user_id: line_user_id)
-          LineApi.push_message(line_user_id, '登録ありがとう！')
+          ActiveRecord::Base.transaction do
+            name = LineApi.get_profile(line_user_id)['displayName']
+            customer = Customer.create!(name: name, line_user_id: line_user_id)
+
+            # 登録完了後のチケットを送る
+            item = Item.find_by(detail_use: "初回登録")
+            request_code = SecureRandom.urlsafe_base64(30)
+            tickets = GajoenApi.issue_tickets(item.brand_id, item.item_id, request_code)
+            ticket = Ticket.create!(create_ticket_prams(tickets, customer.id, request_code))
+            message = "登録ありがとうございました！チケットはこちらになります\n" + tickets['url']
+            LineApi.push_message(line_user_id, message)
+          rescue
+            false
+          end
         end
       end
     }
     head :ok
   end
+
+  private
+
+  def create_ticket_prams(params, id, request_code)
+    {
+      ticket_code:      params['code'],
+      request_code:     request_code,
+      url:              params['url'],
+      status:           params['status'],
+
+      available_begin:  Time.at(params['available_begin']),
+      available_end:    Time.at(params['available_end']),
```

Comment:
[nits]
細かいですが、改行がいらないと思ったのと、`Time.zone.at`を使うように修正してほしいです！

```suggestion
      available_begin:  Time.zone.at(params['available_begin']),
      available_end:    Time.zone.at(params['available_end']),
```

・`Time.at`: railsが動いているサーバーのシステム時間を使う
・`Time.zone.at`: rails内で設定している時刻を使う
という違いがあって、例えばUTCのタイムゾーンで動いているサーバーと日本のタイムゾーンで動いているサーバーで挙動が同じになるように、`Time.zone.at`を使うことが一般的です